.github/workflows/jekyll-gh-pages-avatar.yml

```diff
@@ -30,21 +30,46 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: main
+
       - name: Clone autils repo
         uses: actions/checkout@v3
         with:
           repository: avocado-framework/autils
           path: resources/autils
           ref: main
-      - name: Copy autils metadate to _data dir
+
+      - name: List files in autils archive directory
+        run: ls resources/autils/metadata/autils/archive/
```

Comment:
This command is focused only to archive utility, when we will add more utils to autils this command won't work. This needs to use `ls -R resources/autils/metadata/autils/`

Reply:
I see, thanks for the explanation.